add new test for IntegriVideo project

AnnSchiretz opened the pull request.

AnnSchiretz committed:
add new test for IntegriVideo project

dzmitryrak commented on src/test/java/IntegriVideo/CheckWorkButtonCopyTest.java:
```diff
@@ -0,0 +1,23 @@
+public class CheckWorkButtonCopyTest extends SettingsForTests {
+    @Test
+    public void clickCopyButton() throws IOException, UnsupportedFlavorException {
+        String textOnPage = driver.findElement(By.xpath("//code")).getText();
```
АГОНЬ

dzmitryrak commented on src/test/java/IntegriVideo/CheckWorkInviteButtonTest.java:
```diff
@@ -0,0 +1,25 @@
+public class CheckWorkInviteButtonTest extends SettingsForTests {
+    @Test
+    public void clickInviteButton() throws IOException, UnsupportedFlavorException {
+        driver.findElement(By.id("invite-users-to-chat")).click();
```
АГОНЬ

dzmitryrak commented on src/test/java/IntegriVideo/DeleteMessageTest.java:
```diff
@@ -0,0 +1,26 @@
+    @Test
+    public void sendMessageAndDelete() {
+        String message = "Hello";
+        String checkLocator = "//div[contains(text(), 'Hello')]";
```
плохой локатор. Юзать локатор по классу

dzmitryrak commented on src/test/java/IntegriVideo/DeleteMessageTest.java:
```diff
@@ -0,0 +1,26 @@
+        String message = "Hello";
+        String checkLocator = "//div[contains(text(), 'Hello')]";
+        sendMessage(message, checkLocator);
+        driver.findElement(By.xpath("//div[contains(text(), 'Hello')]/../following-sibling::div/span")).click();
```
если одна строка встречается дважды - выноси в переменную

dzmitryrak replied:
трижды

dzmitryrak commented on src/test/java/IntegriVideo/SettingsForTests.java:
```diff
@@ -0,0 +1,35 @@
+
+    @AfterTest
+    public void afterTest() {
+        driver.quit();
```
Вместо BeforeTest и AfterTest использовать BeforeMethod и AfterMethod

dzmitryrak commented on src/test/java/IntegriVideo/LoadingOnceImageTest.java:
```diff
@@ -0,0 +1,23 @@
+public class LoadingOnceImageTest extends SettingsForTests {
+    @Test
+    public void loadingImg() {
+        String pathImg = "/Users/annschirets/IdeaProjects/IntegriVideo/src/test/java/IntegriVideo/w450h4001385925290Camera — копия.png";
```
относительный путь плиз

AnnSchiretz committed:
changed some locator(add more universal locators);
add WebElement in test, where was it needed;
moving file (Camera.png) to folder resource;
